_opensearch/bucket-agg.md

```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
 
 This is especially true if `size` is set to a low number. Because the default size is 10, an error is unlikely to happen. If you don’t need high accuracy and want to increase the performance, you can reduce the size.
 
+## Multi-terms
+
+Similar to the `terms` bucket aggregation, you can also search for multiple terms using the `multi_terms` aggregation.
+
+You can sort by a number of documents. You can also sort by a metric aggregation on a composite key and get the top `n` results. Consider create a search that contains several terms: the number of documents (e.g. 1000), and the number of servers per location that show CPU usage greater than 90%.
+This multi-term query returns the top number of results.
+
+The multi_terms aggregation does consume more memory, so performance will be slower than a `terms` aggregation.
```

Comment:
I suggest changing this sentence to "The multi_terms aggregation does consume more memory than a 'terms' aggregation, so its performance will be slower."